package.json

```diff
@@ -16,6 +16,17 @@
       "types": "./dist/index.d.ts"
     }
   },
+  "files": [
+    "src/*",
+    "dist/*",
+    ".eslintignore",
+    ".eslintrc.js",
+    ".nvmrc",
+    ".prettierignore",
+    ".versionrc.json",
+    "jest.config.js",
+    "!src/tests"
+  ],
```

Comment:
Why not just `dist`? Do we need the rest of it?

Reply:
I don't think so, `dist/` will be enough: [8f5cf2f](https://github.com/ExodusMovement/aptos-sdk-fork/pull/1/commits/8f5cf2f2b8cc3e1255dfa6844877445e0c78a614)